export for chromium:

re2::FilteredRE2::AllMatches(re2::StringPiece const&, std::vector<int, std::allocator<int> > const&, std::vector<int, std::allocator<int> >*) const'

--- re2-20130115/libre2.symbols~	2013-03-16 14:28:54.000000000 +0200
+++ re2-20130115/libre2.symbols	2013-03-16 14:32:44.874122764 +0200
@@ -10,6 +10,7 @@
 		_ZlsRSoRKN3re211StringPieceE;
 		# re2::FilteredRE2*
 		_ZN3re211FilteredRE2*;
+		_ZNK3re211FilteredRE210AllMatchesERKNS_11StringPieceERKSt6vectorIiSaIiEEPS6_;
 	local:
 		*;
 };
